week-2-wide-perspective/w02e05-simple-server-side/src/logger.ts

```diff
@@ -0,0 +1,19 @@
+import { createLogger, format, transports } from 'winston'
+
+export const logger = createLogger({
```

Comment:
Generalnie logger masz ogarnięty dość dobrze. Brakuje tylko kreowania  tzw. `childLoggera`. 

https://github.com/winstonjs/winston#creating-child-loggers

Chodzi o to żeby sobie robić nowego loggera, z "kontekstem" tam gdzie go będziesz używał.

Więcej o tym będziesz miał w samym `Nest.js`:
https://docs.nestjs.com/techniques/logger#using-the-logger-for-application-logging

Tam jest to nazwane "context logging"

Idea jest następująca. 
Np. jeśli potrzebujesz loggera w "auth.service"
Tworzysz z tego loggera, tzw. childLogger z przedrostkiem `"auth.service"` - dzięki temu, będzie wiedział dokładnie "skąd" pochodzą informacje które logujesz. (od razu złapiesz "kontekst").